package.json

```diff
@@ -85,10 +85,12 @@
     "classnames": "^2.2.6",
     "dotenv-webpack": "^1.5.7",
     "json-server": "^0.14.0",
+    "moment": "^2.23.0",
```

Comment:
are we using both `moment` and `react-moment`? if not, please remove the unused one